src/cow-react/pages/LimitOrders/styled.ts

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components/macro'
+import { MEDIA_WIDTHS } from 'theme'
+
+export const PageWrapper = styled.div`
+  width: 100%;
```

Comment:
please, no CSS in pages either

Reply:
should be in a dumb component. Also, pls make cosmos for the widgets too

If you prefer you can do these in follow up PRs. 

https://github.com/cowprotocol/cowswap/pull/1127

Reply:
I think this should also be open for a discussion for now and we can decide later on